src/libponyc/expr/call.c

```diff
@@ -74,6 +74,32 @@ bool method_check_type_params(pass_opt_t* opt, ast_t** astp)
   return true;
 }
 
+static bool void_star_param(ast_t* param_type, ast_t* arg_type)
```

Comment:
This seems to be a duplicate of what we have in `ffi.c`, perhaps it would be better to call that directly?

Reply:
@ergl Yes -- I for now just c&p-ed. Stylistically, should I simply add a forward decl in ffi.c or try to stuff that function into an extra header, included by both?

Reply:
I think it's fine to add `void_star_param` to `ffi.h` and include it here. I don't think there's a strict rule for what we export in headers in the compiler. There's precedent of exporting utility functions in some places (see `expr/literal.h` for example).